rllib/algorithms/dreamerv3/tf/dreamerv3_tf_rl_module.py

```diff
@@ -12,18 +12,14 @@
 tf1, tf, _ = try_import_tf()
 
 
-class DreamerV3TfRLModule(DreamerV3RLModule, TfRLModule):
+class DreamerV3TfRLModule(TfRLModule, DreamerV3RLModule):
```

Comment:
Can we add a short explanation to all of this that the TfRLModule should be the first and the Mixin should follow it because o.w the setup() will get called twice?

Reply:
Also why are we not doing the same for PPOTorchRLModule in this PR?